services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusService.kt

```diff
@@ -136,17 +142,157 @@ class HakemusService(
         return updatedHakemus.toResponse()
     }
 
+    @Transactional
+    fun sendHakemus(id: Long, currentUserId: String): Hakemus {
+        println("ADSF sendHakemus: $id, $currentUserId")
```

Comment:
debug logging?

Reply:
Yes, thank you. I removed it and added a check for `println` to my git hooks.

Reply:
That sounds useful. I will have to look into this "own global hook" thing.